src/Votes.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+
+const Vote = ({ article }) => {
+    const currVotes = article.votes
```

Comment:
This is not quite how we handle optimistic rendering. We should have separate state and add this to the votes we have to show the change